storage/metrics/metrics.go

```diff
@@ -0,0 +1,35 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// namespace is the leading part of all published metrics for the Storage service.
+const namespace = "storage"
+
+// StorageMetrics wrap all collectors of storage-related metrics.
+type StorageMetrics struct {
+	DefaultLabels *prometheus.Labels
```

Comment:
Not a huge fan of `DefaultLabels` because it implies these could/should get overwritten. I believe the intent from the beta was more to support configurable _global_ labels (i.e. to separately label different nodes in a cluster)